src/App.tsx

```diff
@@ -1,24 +1,197 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
+import { TodoCreate } from './components/TodoCreate';
+import { TodoInfo } from './components/TodoInfo';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = '6757';
+
+enum SortType {
+  completed,
+  active,
+  all,
+}
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>();
+  const [selectedForm, setSelectedForm] = useState(SortType.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [countComplited, setCountComplited] = useState(false);
+  const [countNotComplited, setCountNotComplited] = useState(false);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const setBlock = {
+    setCountComplited,
+    setCountNotComplited,
+  };
+
+  const statusComplited = {
+    countComplited,
+    countNotComplited,
+    todosFromServer,
+  };
```

Comment:
don't create objects here. it's better to propagate every variable as separate property